src/cpu/aarch64/matmul/acl_matmul.cpp

```diff
@@ -125,6 +125,12 @@ status_t acl_matmul_t::pd_t::init(engine_t *engine) {
     CHECK(acl_post_ops.init(engine, attr_.post_ops_, dst_md_, act_info,
             amp_.gemm_info.accumulate() ? 1 : 0));
     amp_.gemm_info.set_activation_info(act_info);
+    // for fp16 matmul the default is to accumulate into fp32 in ACL
```

Comment:
I'm not sure this comment is useful, for f16 matmul, we explicitly set the mode in both cases anyway